### PR TITLE
Add Andy header and empty lines at the end

### PR DESCRIPTION
**Pull request by martinmladenov**
Added the following header before the code that runs Andy:
```
            */(##%#*.
            //**(/////(/*/*/.
         ,/*****,,,,,,,,,,***/#(
       ,#***,,,,,,,,,,,,,*,***//%
       (/***,,,.......,,,,,,**//(
       %/***,,............,,,*///(
       (#****,,.........,*##/,/(/(
       (((*,**,,(,(/*..,#/*%#/#//###
      **#/**,,,,,***,,,,/*,,,*,,/#((
     ,,,,/***,.....,,*.,*(*,,,,*/((
       ,,.,(**,...*,*/,,(%#/***#(#/
         ,,/#/**//*,**.(*///(#*/(.
           /(/**,*/#.. . ,*//*,##
            *(((/,..,,,,*/#**.%(
         %&*/*#,(#,....,,,,,,,#
        %%%%%////&%*.**.*,**(@@&%#
       %%#%%%%%#//*/%&&&%&%(#@@&&&%(%&%/
    #%%%%%%%%%%%%%///**/(#&#&@@@&%%%@/%&
 ##%%%%%%%%%%%%%%%&&%//*/%@@(@&%#@@@%&&& 
#########################################
#                                       #
#                A N D Y                #
#                                       #
#########################################
```

Also, added a few extra lines after the Andy output and at the end.

### CODE REVIEW

File: src/main/java/com/github/cse1110/andy/AndyMojo.java [hunk 1]
```diff
@@ -34,6 +34,38 @@
 
     @Override
     public void execute() {
+        System.out.println(
```
Comment (mauricioaniche) [inline src/main/java/com/github/cse1110/andy/AndyMojo.java:37]: Extract this to a method printHeader() or something

**Review by mauricioaniche — CHANGES_REQUESTED** [review]
Minor change

**Comment by mauricioaniche** [comment]
Also, I was thinking more of just writing "Andy" in three or four lines, like other console apps do. I feel the full ascii art is a bit too big.

**Comment by martinmladenov** [comment]
Should I keep the rectangle with the Andy text and remove everything above it like this?

```
#########################################
#                                       #
#                A N D Y                #
#                                       #
#########################################
```

**Comment by mauricioaniche** [comment]
I was thinking of something like this: http://www.patorjk.com/software/taag/#p=display&f=Doom&t=Andy

WDYT?

**Comment by martinmladenov** [comment]
I agree, this one is better! I'll change it in the code.

**Comment by martinmladenov** [comment]
What do you think about those styles:
Big:
```
                     _       
     /\             | |      
    /  \   _ __   __| |_   _ 
   / /\ \ | '_ \ / _` | | | |
  / ____ \| | | | (_| | |_| |
 /_/    \_\_| |_|\__,_|\__, |
                        __/ |
                       |___/ 
```
Doom (the one you suggested):
```
  ___            _       
 / _ \          | |      
/ /_\ \_ __   __| |_   _ 
|  _  | '_ \ / _` | | | |
| | | | | | | (_| | |_| |
\_| |_/_| |_|\__,_|\__, |
                    __/ |
                   |___/ 
```
Standard:
```
     _              _       
    / \   _ __   __| |_   _ 
   / _ \ | '_ \ / _` | | | |
  / ___ \| | | | (_| | |_| |
 /_/   \_\_| |_|\__,_|\__, |
                      |___/ 
```
Star Wars:
```
     ___      .__   __.  _______  ____    ____ 
    /   \     |  \ |  | |       \ \   \  /   / 
   /  ^  \    |   \|  | |  .--.  | \   \/   /  
  /  /_\  \   |  . `  | |  |  |  |  \_    _/   
 /  _____  \  |  |\   | |  '--'  |    |  |     
/__/     \__\ |__| \__| |_______/     |__|     
```
Roman:
```
      .o.                         .o8              
     .888.                       "888              
    .8"888.     ooo. .oo.    .oooo888  oooo    ooo 
   .8' `888.    `888P"Y88b  d88' `888   `88.  .8'  
  .88ooo8888.    888   888  888   888    `88..8'   
 .8'     `888.   888   888  888   888     `888'    
o88o     o8888o o888o o888o `Y8bod88P"     .8'     
                                       .o..P'      
                                       `Y8P'       
                                                   
```
Univers:
```
                                                         
       db                               88               
      d88b                              88               
     d8'`8b                             88               
    d8'  `8b      8b,dPPYba,    ,adPPYb,88  8b       d8  
   d8YaaaaY8b     88P'   `"8a  a8"    `Y88  `8b     d8'  
  d8""""""""8b    88       88  8b       88   `8b   d8'   
 d8'        `8b   88       88  "8a,   ,d88    `8b,d8'    
d8'          `8b  88       88   `"8bbdP"Y8      Y88'     
                                                d8'      
                                               d8'       
```

I personally like either Big or Standard more than Doom. What do you think?

**Comment by mauricioaniche** [comment]
the first three! You can pick!

**Pull request merged**